SPECS/intel-pf-bb-config/intel-pf-bb-config.spec

```diff
@@ -19,7 +22,8 @@ accessing their configuration space and setting parameters via MMIO.
 %autosetup -n pf-bb-config-%{version}
 
 %build
-./build.sh
+sed -i "s/#VERSION_STRING#/v%{version}-0-gf9206ad/g" config_app.c
```

Comment:
why do we need this? maybe worth adding a comment

Reply:
It's directly from the build.sh script